fontir/src/error.rs

```diff
@@ -81,6 +81,12 @@ pub enum Error {
         n: usize,
         size_n: usize,
     },
+    #[error("Tag '{0}' is not a known axis")]
+    UnknownAxis(Tag),
+    // this stinks but there are so many one-off error conditions that maybe
+    // only occur in one of the source formats that trying to be exhaustive here feels bad
+    #[error("{0}")]
+    Custom(String),
```

Comment:
I suggest two fairly slightly more specific errors instead of just custom, something like:

```rust
// Look ma, I replace UnknownAxis as well!
#[error("Unknown {0}: {1}"]
UnknownEntry(&'static str, String),
#[error("Invalid {0}: {1}"]
InvalidEntry(&'static str, String),
```